src/components/site/header/HeaderProgress.jsx

```diff
@@ -1,40 +1,63 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import last from 'lodash/last';
 import CustomIcon from '../icons/CustomIcon';
 import LinearProgress from '../linearProgress';
 import LinearProgressMarker from '../linearProgress/LinearProgressMarker';
 
 class HeaderProgress extends React.PureComponent {
+  getDisplayValue = (currentPage, totalPages) =>
```

Comment:
`displayValue` will show the progress percentage in relation to the entire investigation, even when the progress marker is inside of a specific section's progress bar.